torch/optim/lr_scheduler.py

```diff
@@ -110,7 +110,7 @@ def print_lr(self, is_verbose, group, lr, epoch=None):
                 print('Adjusting learning rate'
                       ' of group {} to {:.4e}.'.format(group, lr))
             else:
-                print('Epoch {:5d}: adjusting learning rate'
+                print('Epoch {:5f}: adjusting learning rate'
```

Comment:
This would lead to many digits after the `.` in all cases. The print would look weird I think a bit un-expected I think.